index.html

```diff
@@ -1,19 +1,275 @@
+                minlength=2
+              />
+            </fieldset>
+
+            <fieldset>
+              <label for="last_name">Last Name *</label>
+              <input
+                type="text"
+                name="last_name"
+                id="last_name"
+                required=true
+                minlength=2
+              />
+            </fieldset>
+          </div>
+          <div class="address_field">
+            <fieldset>
+              <label for="address">Address *</label>
+              <input
+                type="text"
+                name="address"
+                id="address"
+                required=true
+                minlength=2
+              />
+              <label for="address_two">Address 2</label>
+              <input type="text" name="address_two" id="address_two" />
+            </fieldset>
+          </div>
+
+          <div class="city_postcode">
+            <fieldset>
+              <label for="city">City *</label>
+              <select name="city" id="city" required=true> 
+                <option value="Select your city">Select your city...</option>
+                <option value="Bristol">Bristol</option>
+                <option value="Cardiff">Cardiff</option>
+                <option value="Dorset">Dorset</option>
+                <option value="Enfield">Enfield</option>
+                <option value="Edinburgh">Edinburgh</option>
+                <option value="Manchester">Manchester</option>
+                <option value="Liverpool">Liverpool</option>
+                <option value="London">London</option>
+              </select>
+            </fieldset>
+
+            <fieldset>
+              <label for="postcode">Postcode *</label>
+              <input
+                type="text"
+                name="postcode"
+                id="postcode"
+                required=true
+                minlength=6
+              />
+            </fieldset>
+          </div>
+
+          <p class="select_colour">Select a colour</p>
+
+          <div class="colours_terms">
+            <div class="orange">
+              <input id="orange" type="radio" name="colour" value="orange" />
+              <label for="orange">Karma Orange</label>
+            </div>
+
+            <div class="grey">
+                <input id="grey" type="radio" name="colour" value="grey" />
+              <label for="grey">Space Grey</label>
+            </div>
+              
+            
+
+            <div class="terms">
+              <input id="terms" type="checkbox" name="terms" />
+              <label for="terms">By placing your order you agree to Karma's </label>
+               <label for="terms" class="terms_orange">Terms and
+                Conditions.</label> <label for="terms">*</label>
+            </div>
+          </div>
+          </div>
+          </div>
+          </section>
+        </div>
+      </div>
+      <button class="order_button">Place my order</button>
```

Comment:
Normally when submitting a form we use a submit button. You can turn a button into a submit button using type="submit".

Reply:
I corrected it using "submit" button this time.